test/intl402/Temporal/TimeZone/non-canonical-africa.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2022 André Bargull. All rights reserved.
```

Comment:
Out of curiosity what's the reason the non-canonical time zone name tests can't be in one file?

Reply:
Wouldn't all the files have to be skipped anyway if CLDR canonicalization was used?

We could also assert that the ID is equal to either the expected value for CLDR or IANA, but in practice that probably doesn't tell us anything more than your second option (only testing that the non-canonical name is allowed.)

Reply:
> Wouldn't all the files have to be skipped anyway if CLDR canonicalization was used?

Not necessarily all files, for example all links in "non-canonical-brazil.js" are exactly the same in IANA and CLDR.